src/standard/standard.android.ts

```diff
@@ -50,13 +56,19 @@ export class StripeConfig extends StripeConfigCommon {
   }
 }
 
+
 export class StripeCustomerSession {
   native: com.stripe.android.CustomerSession;
 
   constructor() {
-    com.stripe.android.CustomerSession.initCustomerSession(createKeyProvider());
+    StripeConfig.shared().initPaymentConfiguration();
+    com.stripe.android.CustomerSession.initCustomerSession(this._getContext(), createKeyProvider());
     this.native = com.stripe.android.CustomerSession.getInstance();
   }
+
+  private _getContext(): android.content.Context {
```

Comment:
Minor stylistic point. Personally, I prefer
`private get context(): android.context.Context {}`

which I think looks better at the call site: `this.context`.